recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -1,81 +1,75 @@
 # ----------------------------------------------------------------------------
 
 # More complicated plugins are moved seperate includes
 
+include include/bluetooth.inc
+include include/cobalt.inc
 include include/compositor.inc
+include include/dhcpserver.inc
+include include/dictionary.inc
 include include/dialserver.inc
 include include/displayinfo.inc
+include include/filetransfer.inc
 include include/firmwarecontrol.inc
+include include/inputswitch.inc
 include include/ioconnector.inc
+include include/languageadministrator.inc
 include include/network.inc
-include include/power.inc
+include include/performancemonitor.inc
 include include/playerinfo.inc
+include include/power.inc
+include include/processcontainers.inc
+include include/processmonitor.inc
 include include/remotecontrol.inc
+include include/resourcemonitor.inc
+include include/snapshot.inc
+include include/svalbard.inc
+include include/systemcommands.inc
+include include/systemdconnector.inc
+include include/timesync.inc
+include include/volumecontrol.inc
+include include/webproxy.inc
+include include/webserver.inc
+include include/webshell.inc
+include include/wifi.inc
 
 # ----------------------------------------------------------------------------
 
 WPEFRAMEWORK_LOCATIONSYNC_URI ?= "http://jsonip.metrological.com/?maf=true"
-PLUGIN_WEBSERVER_PORT ?= "8080"
-PLUGIN_WEBSERVER_PATH ?= "/var/www/"
 
 # ----------------------------------------------------------------------------
 
-PACKAGECONFIG ?= " ${@bb.utils.contains('MACHINE_FEATURES', 'bluetooth', 'bluetoothcontrol', '', d)} \
+PACKAGECONFIG ??= "\
+    ${@bb.utils.contains('MACHINE_FEATURES', 'bluetooth', 'bluetoothcontrol', '', d)} \
     ${@bb.utils.contains('MACHINE_FEATURES', 'bluetooth', 'bluetoothremote', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', 'compositor', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'systemdconnector', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'thunder', 'network', '', d)} \
     ${@bb.utils.contains('MACHINE_FEATURES', 'wifi', 'network wifi', '', d)} \
     ${@bb.utils.contains('STREAMER_DISTRO_PACKAGE_AVAILABLE', 'True', 'streamer', '', d)} \
-    dhcpserver dictionary ioconnector remote remote-devinput systemcommands timesync webserver"
-
-PACKAGECONFIG_append_brcm = " displayinfo snapshot volumecontrol"
-PACKAGECONFIG[bluetoothcontrol] = "-DPLUGIN_BLUETOOTH=ON \
-    -DPLUGIN_BLUETOOTH_AUTOSTART=true \
-    ,-DPLUGIN_BLUETOOTH=OFF,,bluez5"
-PACKAGECONFIG[bluetoothremote] = "-DPLUGIN_BLUETOOTHREMOTECONTROL=ON \
-    -DPLUGIN_BLUETOOTHREMOTECONTROL_AUTOSTART=true \
-    ,-DPLUGIN_BLUETOOTHREMOTECONTROL=OFF,"
-PACKAGECONFIG[dhcpserver] = "-DPLUGIN_DHCPSERVER=ON,-DPLUGIN_DHCPSERVER=OFF,"
-PACKAGECONFIG[dictionary] = "-DPLUGIN_DICTIONARY=ON,-DPLUGIN_DICTIONARY=OFF,"
-PACKAGECONFIG[dsgcc_client] = "-DPLUGIN_DSGCCCLIENT=ON,,broadcom-refsw"
-PACKAGECONFIG[dsresolution] = "-DPLUGIN_DSRESOLUTION=ON,,devicesettings"
```

Comment:
Can we keep that for backward compatibility please? Maybe a separate plugins_depracated.inc file is ok

Reply:
done